cmd/docker-driver/driver.go

```diff
@@ -65,16 +72,29 @@ func (d *driver) StartLogging(file string, logCtx logger.Info) error {
 		return fmt.Errorf("logger for %q already exists", file)
 	}
 	d.mu.Unlock()
+	folder := fmt.Sprintf("/var/log/docker/%s/", logCtx.ContainerID)
+	logCtx.LogPath = filepath.Join(folder, "json.log")
 
-	if logCtx.LogPath == "" {
-		logCtx.LogPath = filepath.Join("/var/log/docker", logCtx.ContainerID)
-	}
-	if err := os.MkdirAll(filepath.Dir(logCtx.LogPath), 0755); err != nil {
+	if err := os.MkdirAll(folder, 0755); err != nil {
```

Comment:
Empty folders will still be created and removed even with `no-file = true`

Reply:
Yeah do you think this is a problem ? let me see how easy this could be fixed.